rspec-trunk-flaky-tests/lib/trunk_spec_helper.rb

```diff
@@ -187,19 +187,24 @@ def close(_notification)
 
   # trunk-ignore(rubocop/Metrics/CyclomaticComplexity,rubocop/Metrics/AbcSize,rubocop/Metrics/MethodLength)
   def add_test_case(example)
+    return if ENV['KNAPSACK_PRO_TEST_EXAMPLE_DETECTOR'] == 'true'
```

Comment:
[Undocumented](https://github.com/search?type=code&q=%22KNAPSACK_PRO_TEST_EXAMPLE_DETECTOR%22). I'm assuming you found this by hand? If this works, let's add a canary for it